scripts/formapplications/prompt-rest/prompt-rest-shell.svelte

```diff
@@ -87,10 +87,27 @@
     const timeChanges = lib.getTimeChanges(restType === "longRest");
     await game.time.advance(timeChanges.restTime);
 
-    SocketHandler.emit(SocketHandler.PROMPT_REST, {
-      userActors: [...configuration],
-      restType
-    })
+    let restText = (restType === "longRest") ? game.i18n.format("REST-RECOVERY.Chat.CouldNotLongRest") : game.i18n.format("REST-RECOVERY.Chat.CouldNotShortRest");
+
+    configuration.forEach((element) => {
+      if (getProperty(game.actors.get(element.split("-")[1]), `flags.dae.RR5EPrevent${restType}`)) {
+          configuration.delete(element);
+          ChatMessage.create({
+            content: `<br>${game.actors.get(element.split("-")[1]).name} ${restText}`,

```

Comment:
I'd recommend changing it to this, based on the localization changes above:
```js
content: "<p>" + localize("REST-RECOVERY.Chat.CouldNot" + (restType === "longRest" ? "LongRest" : "ShortRest"), { actor_name: game.actors.get(element.split("-")[1]).name }) + "</p>"
```
`localize()` is an utility function that the UI library has created that combines `game.i18n.localize()` and `game.i18n.format()`